metrics/server.go

```diff
@@ -20,8 +21,12 @@ func Start(addr string, logger *zap.Logger) *Server {
 
 	// Create the HTTP server with the custom ServeMux as the handler
 	server := &http.Server{
-		Addr:    addr,
-		Handler: mux,
+		Addr:              addr,
+		Handler:           mux,
+		ReadHeaderTimeout: 2 * time.Second,
+		ReadTimeout:       1 * time.Second,
```

Comment:
I would say that timeouts for reading are bit to aggressive here. 

I would set both to `5seconds`. Mostly becouse I saw such values used in:
- https://blog.cloudflare.com/exposing-go-on-the-internet/
- and https://blog.cloudflare.com/the-complete-guide-to-golang-net-http-timeouts/

Reply:
done